.github/workflows/build.yml

```diff
@@ -199,6 +199,24 @@ jobs:
         APPLE_TEAMID: ${{ secrets.APPLE_TEAMID }}
         CERTIFICATE_MACOS_P12_BASE64: ${{ secrets.CERTIFICATE_MACOS_P12_BASE64 }}
         CERTIFICATE_MACOS_P12_PASSWORD: ${{ secrets.CERTIFICATE_MACOS_P12_PASSWORD }}
+    
+    - name: Package AppImage
+      if: startsWith(runner.os, 'linux')
+      run: |
+        ZIP_FILE=`ls ./dist/ -1 |grep zip`
+        unzip ./dist/$ZIP_FILE
+        wget https://github.com/linuxdeploy/linuxdeploy/releases/download/continuous/linuxdeploy-x86_64.AppImage
+        chmod +x linuxdeploy-x86_64.AppImage
+        wget https://github.com/AppImage/AppImageKit/releases/download/continuous/appimagetool-x86_64.AppImage
+        chmod +x appimagetool-x86_64.AppImage
+        cp ./activitywatch/aw-server-rust/static/static/logo.png ./activitywatch/activitywatch.png
+        echo '#!/bin/sh
+        DIR="$(dirname "$(readlink -f "${0}")")"
+        "${DIR}"/aw-qt'>activitywatch/AppRun
+        ./linuxdeploy-x86_64.AppImage --appdir activitywatch --executable ./activitywatch/aw-qt --output appimage
+        APPIMAGE_FILE=`ls -1 |grep AppImage|grep -i ActivityWatch`
+        cp $APPIMAGE_FILE ./dist/
+
```

Comment:
Example name of built file `ActivityWatch-9f23a1b-x86_64.AppImage`  
Do you think it should be renamed to `activitywatch-x86_64.AppImage` to ensure consistency with names of other packages?

Reply:
@ErikBjare   
Could you also answer me on this one?

Reply:
I think  `activitywatch-{version_or_commithash}-linux-x86_64.zip` is good, to keep naming of release artifacts consistent.

Reply:
I used
`activitywatch-linux-x86_64.AppImage`  
(without version/hash, because all the version should have the same name to be easily swappable)